scalr/data_ingestion_pipeline.py

```diff
@@ -117,7 +123,7 @@ def preprocess_data(self):
                              self.sample_chunksize)
             # Transform on train, val & test split.
             for split in ['train', 'val', 'test']:
-                preprocessor.process_data(path.join(datapath, split),
+                preprocessor.process_data(read_data(path.join(datapath, split)),
```

Comment:
let's make a new variable for data and pass that as a parameter.

Reply:
done